src/patternfly/components/EmptyState/empty-state.scss

```
@@ -17,40 +17,49 @@
   --#{$empty-state}--m-xl__icon--FontSize: #{pf-size-prem(100px)};
 
+  // Status variant icon color
+  --#{$empty-state}--m-danger__icon--Color: var(--pf-t--global--icon--color--status--danger--default);
+  --#{$empty-state}--m-warning__icon--Color: var(--pf-t--global--icon--color--status--warning--default);
+  --#{$empty-state}--m-success__icon--Color: var(--pf-t--global--icon--color--status--success--default);
+  --#{$empty-state}--m-info__icon--Color: var(--pf-t--global--icon--color--status--info--default);
+  --#{$empty-state}--m-custom__icon--Color: var(--pf-t--global--icon--color--status--custom--default);
+
   // Title text
-  --#{$empty-state}__title-text--FontFamily: var(--#{$pf-global}--FontFamily--heading);
-  --#{$empty-state}__title-text--FontSize: var(--#{$pf-global}--FontSize--xl);
-  --#{$empty-state}__title-text--FontWeight: var(--#{$pf-global}--FontWeight--normal);
-  --#{$empty-state}__title-text--LineHeight: var(--#{$pf-global}--LineHeight--md);
-  --#{$empty-state}--m-xs__title-text--FontSize: var(--#{$pf-global}--FontSize--md);
-  --#{$empty-state}--m-xl__title-text--FontSize: var(--#{$pf-global}--FontSize--4xl);
-  --#{$empty-state}--m-xl__title-text--LineHeight: var(--#{$pf-global}--LineHeight--sm);
+  --#{$empty-state}__title-text--FontFamily: var(--pf-t--global--font--family--heading);
+  --#{$empty-state}__title-text--FontSize: var(--pf-t--global--font--size--heading--md);
+  --#{$empty-state}__title-text--FontWeight: var(--pf-t--global--font--weight--heading);
+  --#{$empty-state}__title-text--LineHeight: var(--pf-t--global--font--line-height--heading);
+  --#{$empty-state}--m-xs__title-text--FontSize: var(--pf-t--global--font--size--heading--xs);
+  --#{$empty-state}--m-xl__title-text--FontSize: var(--pf-t--global--font--size--heading--2xl);
+  --#{$empty-state}--m-xl__title-text--LineHeight: var(--pf-t--global--font--line-height--heading);
 
   // Body
-  --#{$empty-state}__body--MarginTop: var(--#{$pf-global}--spacer--md);
-  --#{$empty-state}__body--Color: var(--#{$pf-global}--Color--200);
-  --#{$empty-state}--body--FontSize: var(--#{$pf-global}--FontSize--md);
-  --#{$empty-state}--m-xs__body--FontSize: var(--#{$pf-global}--FontSize--sm);
-  --#{$empty-state}--m-xs__body--MarginTop: var(--#{$pf-global}--spacer--md);
-  --#{$empty-state}--m-xl__body--FontSize: var(--#{$pf-global}--FontSize--xl);
-  --#{$empty-state}--m-xl__body--MarginTop: var(--#{$pf-global}--spacer--lg);
+  --#{$empty-state}__body--MarginTop: var(--pf-t--global--spacer--md);
+  --#{$empty-state}__body--Color: var(--pf-t--global--text--color--subtle);
+  --#{$empty-state}--body--FontSize: var(--pf-t--global--font--size--body--lg);
+  --#{$empty-state}--m-xs__body--FontSize: var(--pf-t--global--font--size--body);
+  --#{$empty-state}--m-xs__body--MarginTop: var(--pf-t--global--spacer--md);
+  --#{$empty-state}--m-xl__body--FontSize: var(--pf-t--global--font--size--body--lg);
```

```suggestion
  --#{$empty-state}--m-xl__body--FontSize: var(--pf-t--global--font--size--heading--md);
```

Design spec shows `20px` font-size here. Should it be `--pf-t--global--font--size--heading--md`?

---

@lboehling I suspect we still want a body text here? Can you confirm?

---

(@lboehling said to use Body Large - a change in Figma wasn't merged yet)